onnxruntime/core/graph/ep_api_types.cc

```diff
@@ -113,12 +113,33 @@ Status EpNode::Create(const Node& node, const EpGraph* ep_graph,
 
   if (node_attrs.size() > 0) {
     ep_node_attributes.reserve(node_attrs.size());
+    std::unordered_map<std::string_view, std::unique_ptr<OrtValue>> tensor_attribute_values;
 
```

Comment:
`Constant` op and `ConstantOfShape` op both support 'TENSOR' attribute.
For `ConstantOfShape`, it even supports if 'TENSOR' attribute is not provided, it defaults to a tensor of value 0 and datatype float32. Will add the support here as well.